src/pages/[platform]/build-a-backend/data/customize-authz/public-data-access/index.mdx

```diff
@@ -44,16 +44,81 @@ const schema = a.schema({
 });
 ```
 
-## Add public authorization rule using IAM authentication
-
-You can also override the authorization provider. In the example below, `iam` is specified as the provider which allows you to use an "Unauthenticated Role" from the Cognito identity pool for public access instead of an API key. Your Auth resources defined in `amplify/auth/resource.ts` generates scoped down IAM policies for the "Unauthenticated role" in the Cognito identity pool automatically.
```

Comment:
Did we mean to remove this? I think this text is still supposed to be here, but it looks like two PRs modified the same area so git got confused


I think these PRs changed this file:
- https://github.com/aws-amplify/docs/pull/7056/files#diff-6f342380a22198bf8fec5cd5aa52c4b162dda1de5d169734ed9dc4c7b8bbebbd
- https://github.com/aws-amplify/docs/pull/7080/files#diff-6f342380a22198bf8fec5cd5aa52c4b162dda1de5d169734ed9dc4c7b8bbebbd

Reply:
I think yes. Here's the comparison between the file currently on [`main`](https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx?plain=1) and the [file in this PR](https://github.com/aws-amplify/docs/blob/1996660c7b67f73394850be8b73c86a8b28cedd0/src/pages/%5Bplatform%5D/build-a-backend/data/customize-authz/public-data-access/index.mdx?plain=1).

Reply:
Ah I see, sorry about that. I see the text is still there actually within your changes